yt_dlp/extractor/franceculture.py

```diff
@@ -8,19 +8,29 @@
     int_or_none,
 )
 
+import re
+
 
 class FranceCultureIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?franceculture\.fr/emissions/(?:[^/]+/)*(?P<id>[^/?#&]+)'
     _TESTS = [{
-        'url': 'http://www.franceculture.fr/emissions/carnet-nomade/rendez-vous-au-pays-des-geeks',
+        # playlist
+        'url': 'https://www.franceculture.fr/emissions/hasta-dente',
+        'playlist_mincount': 12,
+        'playlist_maxcount': 12,
```

Comment:
Don't set maxcount
test will break if new videos are added in future

Reply:
This is set on purpose, it's a radio series that has only 12 episodes, but there is a risk in the future if the markup changes slightly that the regex can get other entries from the page that are not related to the topic you're trying to download (there's a section of "you could also listen to these other podcasts" on every page).

Reply:
use `playlist_count` instead of setting both max and min